armi/reactor/blueprints/isotopicOptions.py

```diff
@@ -553,7 +553,7 @@ def eleExpandInfoBasedOnCodeENDF(cs):
             )
 
     elif cs[CONF_XS_KERNEL] in ["", "SERPENT", "MC2v3", "MC2v3-PARTISN"]:
-        elementalsToKeep.update(endf70Elementals)
+        elementalsToKeep.update(endf71Elementals)
```

Comment:
I feel like `SERPENT` can be removed from this list since its not tested here. Additionally, doesn't this remove support of `ENDF/B-VII.0` explicitly? It feels like there should be a setting for which library is being selected OR we should ensure that that changelog notes that this removes the backwards capability.

Reply:
@john-science, any concerns with removing `SERPENT` from this list? We could also just make another conditional that has `SERPENT` default to the `endf70Elementals` .

Having a setting for the library was my original plan, but global nuclides complicates this. John and I have are hoping to implement such a setting once global nuclides are addressed (https://github.com/terrapower/armi/issues/473).

Reply:
I understand the complications of the mixed library options based on that ticket. @drewejohnson originally made this setting, so I think it should be appropriate to remove at this time since there is no use for it. I am OK with keeping it as an option though and adding `endf70Elementals` on that branch or removing `SERPENT` and updating the changelog.

Reply:
Alright Jake, I added a conditional so that VII.0 elementals are used with Serpent.